lib/ansible/modules/git.py

```diff
@@ -1249,11 +1249,11 @@ def main():
         if local_mods:
             # failure should happen regardless of check mode
             if not force:
-                module.fail_json(msg="Local modifications exist in repository (force=no).", **result)
+                module.fail_json(msg="Local modifications exist in the destination directory (force=no).", **result)
```

Comment:
instead of rephrasing, i would just add the specific directory here

also requires a changelog

Reply:
Hi @bcoca 
Thanks for the suggestion.
Changes are done, tested, it now shows the destination on the msg line.
Can you explain how I would do a changelog? I've never done that for this repo before, if you could just help me do that, thanks.

Reply:
Hi @ealmonte32 thanks for the response (and for the PR!). We have documentation on adding changelog entries here: https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs. If you have questions that the docs don't cover, join the #ansible-docs, #ansible-community, or #ansible-dev channels on freenode IRC, we're usually hanging out there.

Reply:
Hey @acozine , thanks for the info/link 😄 
Is this what I needed to do? -> https://github.com/ansible/ansible/pull/72624

Reply:
@acozine @bcoca 
Ok sorry for the delay, it was first time having to add changelog file to already committed PR.
It's done. Let me know if that is all.
Thanks.